app/src/main/java/org/fossasia/openevent/general/di/Modules.kt

```diff
@@ -259,7 +259,7 @@ val networkModule = module {
         val builder = OkHttpClient().newBuilder()
             .connectTimeout(connectTimeout.toLong(), TimeUnit.SECONDS)
             .readTimeout(readTimeout.toLong(), TimeUnit.SECONDS)
-            .addInterceptor(get())
+            .addInterceptor(RequestAuthenticator(get()) as Interceptor)
```

Comment:
Casting is bad

Reply:
Sorry, It is not required now as the response is not nullable now. Updated.